xbmc/platform/win10/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set(SOURCES CPUInfoWin10.cpp
             Win10App.cpp
             MessagePrinter.cpp
             PlatformWin10.cpp
+            WinRtUtil.cpp
```

Comment:
Being a common code for Win32 and UWP, it must be in the `win32` path. I know the name is not much appropriate and somewhat confusing but the `win10` path only contains UWP exclusive code (not used in desktop path).

WinRtUtil name is fine, only move to xbmc/platform/win32

Reply:
ok, done, as well as the other comments.